e2e/src/step-definitions/delegate.steps.ts

```diff
@@ -0,0 +1,47 @@
+import { Given } from '@cucumber/cucumber';
+import { ElementHandle } from 'puppeteer';
+
+import { testDataForInput } from '../classes/test-data-for-input.class';
+import { Pages } from '../page-objects';
+import { TWENTY_SECONDS_TIMEOUT } from '../utils/timing.utils';
+
+Given(/I check who the delegated baker is/, async () => {
+  const delegatedBaker = await Pages.DelegateTab.delegatedBakerName.getText();
+
+  if (delegatedBaker === 'Everstake') {
+    // To avoid "Not allowed" error it will be used another baker address (Ipocamp)
+    testDataForInput.bakerAddress = 'tz1fbumHTEhLMBmtT1GjagbMnhnTD5YZAJh2';
```

Comment:
maybe move this address to constants? If you give them meaningful names, you could aviod comments